week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,45 @@
 // Add comments to explain what this function does. You're meant to use Google!
+/*Math.random() returns a number lower than 1. In this function it returns a
+random integer from 0 to 9 */
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+/*This function joins w1 and w2 and returns the new value   */
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
-    // Look at the test case below to understand what to expect in return 
+  return firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
Interesting approach! It's a good thing that you learnt about the .concat() string method! How could you rewrite this to use the interpolation we learned in class to make this more readable?